docs/configuration/settings.rst

```diff
@@ -480,6 +480,28 @@ In order to replace it by another one:
     multiauth.policy.basicauth.use = myproject.authn.BasicAuthPolicy
 
 
+Mohawk request Auth
+:::::::::::::::::::
+
+`Mohawk <https://mohawk.readthedocs.io/en/latest/index.html>`_ implements the 
+`Hawk authorization scheme <https://github.com/hueniverse/hawk>`_, which 
+allows you to authenticate messages from your application backend to Kinto.
+In order to prevent replay attacks, nonce checking is enforced.  
+See `here <https://github.com/hueniverse/hawk#implementations>` for 
+some implementations of Hawk in different programming languages.
+
+Mohawk auth is enabled by turning on ``kinto.hawk.enabled`` and providing 
+credential values for ``kinto.hawk.id``, ``kinto.hawk.secret``, and
+``kinto.hawk.algo``.  
+
+.. code-block:: ini
+
+    kinto.hawk.enabled = true
+    kinto.hawk.id = 'some client id'
```

Comment:
Arguably this id and secret should be unique per users.

Reply:
Hey @Natim ,
So you think we should use one secret to authenticate all users on a kinto instance? So userid will be used as client ID and the secret will be defined in settings.  Am I misunderstanding that?   
When you say we can "rotate the secret as needed", do we want users to just update the plaintext secret in the settings when they want to, or do we want to provide a way to generate a new secret and rotate it from the CLI?    

thanks!

Reply:
> So you think we should use one secret to authenticate all users on a kinto instance?

No not at all, that's what you've implemented but that's not what we are aiming at.

> So user ID will be used as client ID and the secret will be defined in settings.

I guess at some point we should have an endpoint to trade the account identifiers with a Hawk secret.

The secret will not be defined in the settings, but generated randomly and stored inside the database as a property of the account record.

> When you say we can "rotate the secret as needed"

For now you can ignore that part, I will try to put together a blueprint to explain that in details.